### ISSUE TITLE
[Accessibility]A11y_VSCode_ViewSearch_ScreenReader: ScreenReader does not read the descriptive name for "Clear" button.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52584]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10 / HighSierra10.13.5

**Additional Details:** 
MAS Violated : MAS 1.3.1
ScreenReader Versions : NVDA2018.1.1 / JAWS18.0.5038 / VoiceOver

**Repro Steps:**
1. Launch VS Code while ScreenReader is ON.
2. Navigate to Activity Bar and select "Search"(Cntrl+Shift+F) button.
3. Navigate to "Clear" button above Search text field.

**Actual:**
ScreenReader does not read the clear descriptive name for "Clear" button above Search text field.
It only clears the Search edit field when there are 3 other edit fields as well. Button's name does not justify the functionality and its use.

**Same issue has been observed with JAWS/VoiceOver.**

**Expected:**
ScreenReader should read the clear descriptive name for "Clear" button as "Clear Search" to justify the functionality and its use.

**Recommendations:**
Rename Clear button as "Clear Search" button.
or,
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screenreader users will not understand the functionality and the use of "Clear" button.

**MAS Reference:**
MAS 1.3.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[A11y_VSCode_ViewSearch_ScreenReader_ClearButton.pptx](https://github.com/Microsoft/vscode/files/2123173/A11y_VSCode_ViewSearch_ScreenReader_ClearButton.pptx)

Does this issue occur when all extensions are disabled?: Yes
